compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/res/PainterResources.desktop.kt

```diff
@@ -59,7 +59,7 @@ fun painterResource(
     ResourceLoader.Default
 )
 
-@ExperimentalComposeUiApi
+@Deprecated("Migrate to the compose resources library")
```

Comment:
We don't need to 
1. Remove experimental
2. Deprecate experimental (just remove)

Reply:
We cannot delete these function because they are being used in non experimental public API 🤷 
Example:
```kotlin
//non experimental
fun painterResource(resourcePath: String, loader: ResourceLoader): Painter

//non experimental
private fun rememberSvgResource(
    resourcePath: String, 
    loader: ResourceLoader = ResourceLoader.Default
): Painter

//experimental
internal inline fun <T> useResource(
    resourcePath: String,
    loader: ResourceLoader,
    block: (InputStream) -> T
): T
```

Reply:
Can we make it internal then?